ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/StatementExecutor.java

```diff
@@ -261,6 +261,18 @@ private void handleRunScript(final Command command) {
           ksqlConfig.overrideBreakingConfigsWithOriginalValues(command.getOriginalProperties()),
           overriddenProperties
       );
+      if (!ksqlEngine.hasCapacityForPersistentQueries(
+          0, getPersistentQueryLimit(command))) {
```

Comment:
0 because the queries have already been built on lines 259-263 above, which means they have already been added into the engine's active, persistent query count.